repository-data/development/src/main/resources/hcm-content/content/documents/lks/training-pages.yaml

```diff
@@ -11,7 +11,7 @@
     jcr:mixinTypes: ['hippo:named', 'hippo:versionInfo', 'mix:referenceable']
```

Comment:
Could you remove dev data updates other than the featured content block-related changes from this file for clarity ?